README.md

```diff
@@ -79,10 +79,9 @@ frontend:
 
 Create a workflow (eg: `.github/workflows/labeler.yml` see [Creating a Workflow file](https://help.github.com/en/articles/configuring-a-workflow#creating-a-workflow-file)) to utilize the labeler action with content:
 
-```
+```yaml
 name: "Pull Request Labeler"
-on:
-- pull_request_target
+on: [ pull_request ]
```

Comment:
Hello @JJ ! Thank you for the contribution!
Could you please provide us with the justification for this change?
The `pull_request_target` event allows to add labels to PRs from forks. There is also a warning in the README file about the permissions granted.

Reply:
To be honest, I'll have to dig my memory to understand why... Give me some time.

Reply:
Thank you for the quick response! 
In this case, I am going to close this PR. Please contact us if you have any concerns